lib/book.py

```diff
@@ -443,28 +281,27 @@ def create_xray(self, aConnection, sConnection, info, log=None, abort=None, remo
             return
         if log: log('%s \t\tCreating x-ray...' % datetime.now().strftime('%m-%d-%Y %H:%M:%S'))
         self._write_xray(info, remove_files_from_dir=remove_files_from_dir)
-        return (aConnection, sConnection)
+        return connection
```

Comment:
The reason for this was that if the connection wasn't used for a long enough time, it would stop working and we'd get an error so somewhere along the way, the connection gets closed and reopened. I wasn't sure if the original variable that was sent in would be updated outside of the function that reopened it so I send it back to be safe. My original intention was to test it but I never got around to it.